modules/sql-engine/src/integrationTest/java/org/apache/ignite/internal/sql/engine/ItCreateTableDdlTest.java

```diff
@@ -298,22 +299,9 @@ public void explicitColocationColumnsCaseSensitive() {
     @Test
     public void doNotAllowFunctionsInNonPkColumns() {
         assertThrowsSqlException(
-                Sql.STMT_VALIDATION_ERR,
+                STMT_VALIDATION_ERR,
                 "Functional defaults are not supported for non-primary key columns",
                 () -> sql("create table t (id varchar primary key, val varchar default gen_random_uuid)")
         );
     }
-
-    @Test
-    public void dummyAlterColumnDataType() {
```

Comment:
Why do you drop the test?
Is it a duplicate?

Reply:
yes this test fully covered by tests from ItAlterTableAlterColumnTest